src/backends/kimchi/prover.rs

```diff
@@ -62,6 +62,7 @@ pub struct VerifierIndex {
 // Setup
 //
 
+#[allow(clippy::type_complexity)]
```

Comment:
I guess we should create an issue on removing all the `#[allow.(..` ^^

Reply:
#103

Reply:
remove